udata/search/commands.py

```diff
@@ -69,150 +46,31 @@ def index_model(index_name, adapter, timeout=None):
         qs = qs.exclude(*adapter.exclude_fields)
```

Comment:
this generates error when `udata init` 
`AttributeError: type object 'DatasetSearch' has no attribute 'exclude_fields'`